sql/plsql/PlSqlParser.g4

```diff
@@ -3767,6 +3832,11 @@ dblink
     : database_name ('.' d=id_expression)* ('@' cq=id_expression)?
     ;
 
+// https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/DROP-DATABASE.html
+drop_database
+    : DROP DATABASE
+    ;
+
```

Comment:
The same parse rule already exists: https://github.com/antlr/grammars-v4/blob/ef698f1a02103d7e026ce7b3ea0b4120d4fc487c/sql/plsql/PlSqlParser.g4#L4060-L4063

Please reuse it.